app_integrations/main.py

```diff
@@ -0,0 +1,47 @@
+"""
+Copyright 2017-present, Airbnb Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from app_integrations.config import AppConfig
+from app_integrations.apps.app_base import get_app
+
+
+def handler(event, context):
+    """Main lambda handler use as the entry point
+
+    Args:
+        event (dict): Always empty (for now) event object
+        context (LambdaContxt): AWS LambdaContext object
+    """
+    if event and 'full_run' in event:
+        # TODO: implement support for historical runs via input events
```

Comment:
i think we've traditionally raised an exception when an unsupported/future feature was used?

Reply:
There's really no point in raising the exception right now, since if it _was_ used it just won't have an effect (also note: there there is no way this _could_ be used right now and this is mostly in place so I remember how I wanted to implement this).